var/spack/repos/builtin/packages/multicharge/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Multicharge(CMakePackage):
+    """Electronegativity equilibration model for atomic partial charges"""
+
+    homepage = "https://github.com/grimme-lab/multicharge"
+    url = "https://github.com/grimme-lab/multicharge/releases/download/v0.0.0/multicharge-0.0.0.tar.xz"
+    git = "https://github.com/grimme-lab/multicharge.git"
+
+    maintainers("RMeli", "awvwgk")
```

Comment:
@awvwgk I took the liberty to add you as a maintainer here. If you don't want to be a maintainer, please let me know and I'll remove you from this package.

Reply:
I suppose unless you get an explicit ok you might want to remove @awvwgk from here, but that can be done as the last thing before merging if there's no further response.